src/braket/circuits/instruction.py

```diff
@@ -81,6 +83,24 @@ def target(self) -> QubitSet:
         """
         return self._target
 
+    def adjoint(self) -> Instruction:
+        """Returns an Instruction with the adjoint of this instruction's own operator
+
+        This operation only works on Gate operators.
+
+        Returns:
+            Instruction: A new instruction whose operator is the adjoint of `operator`
+
+        Raises:
+            NotImplementedError: If `operator` is not of type `Gate` or `CompilerDirective`
```

Comment:
Will this be a problem for unitary gates? Will it cause a minefield?

Reply:
Nope, unitaries have well-defined adjoints (and adjoint lists)